Tests/SonarScanner.MSBuild.PreProcessor.Test/WebClientDownloaderTest.cs

```diff
@@ -34,26 +35,27 @@ public class WebClientDownloaderTest
         public void Ctor_SecurityProtocolIsDefault_RemainsDefault()
         {
             // Arrange
-            ServicePointManager.SecurityProtocol = SecurityProtocolType.SystemDefault;
+            var securityProtocolHandlerMock = new Mock<ISecurityProtocolHandler>();
+            securityProtocolHandlerMock.Setup(x => x.SecurityProtocol).Returns(SecurityProtocolType.SystemDefault);
 
-            // Act
-            _ = new WebClientDownloader(null, null, new TestLogger());
+            // Act & Assert
+            _ = new WebClientDownloader(null, null, new TestLogger(), securityProtocolHandlerMock.Object, null, null);
 
-            // Assert
-            ServicePointManager.SecurityProtocol.Should().Be(SecurityProtocolType.SystemDefault);
+            securityProtocolHandlerMock.VerifySet(x => x.SecurityProtocol = SecurityProtocolType.Tls13 | SecurityProtocolType.Tls12 | SecurityProtocolType.Tls11 | SecurityProtocolType.Tls, Times.Never);
         }
 
         [TestMethod]
         public void Ctor_SecurityProtocolIsNotDefault_AllTlsVersionsAreEnabled()
         {
             // Arrange
-            ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls;
+            var securityProtocolHandlerMock = new Mock<ISecurityProtocolHandler>();
+            securityProtocolHandlerMock.Setup(x => x.SecurityProtocol).Returns(SecurityProtocolType.Ssl3);
```

Comment:
please write a comment with the assumption that Ssl13 should not be SystemDefault, at least on the systems we run the tests on (will help reader)

Reply:
I don't understand. How could Ssl3 be SystemDefault?  For Net Framework < 4.6 the default value of the property will be Ssl3 and TLS1.0 but on every framework version the enum values are the same, so I don't understand how the two could be equal?

Reply:
I was suggesting making the implicit explicit.

The below would work better than a comment

```suggestion
            var isNeverDefault = SecurityProtocolType.Ssl3;
            securityProtocolHandlerMock.Setup(x => x.SecurityProtocol).Returns(isNeverDefault);
```